central/convert/storagetov2/vuln_exception_service.go

```diff
@@ -0,0 +1,168 @@
+package storagetov2
+
+import (
+	"github.com/pkg/errors"
+	v2 "github.com/stackrox/rox/generated/api/v2"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/utils"
+)
+
+// VulnerabilityException converts *storage.VulnerabilityRequest to *v2.VulnerabilityException.
+func VulnerabilityException(inp *storage.VulnerabilityRequest) *v2.VulnerabilityException {
+	if inp == nil {
+		return nil
+	}
+
+	out := &v2.VulnerabilityException{
+		Id:          inp.GetId(),
+		Name:        inp.GetName(),
+		TargetState: convertVulnerabilityState(inp.GetTargetState()),
+		Status:      convertRequestStatus(inp.GetStatus()),
+		Expired:     inp.GetExpired(),
+		Requester:   convertUser(inp.GetRequestor()),
+		Approvers:   convertUsers(inp.GetApprovers()),
+		LastUpdated: inp.GetLastUpdated(),
+		Comments:    convertRequestComments(inp.GetComments()),
+		Scope:       convertScope(inp.GetScope()),
+		Cves:        inp.GetCves().GetCves(),
+	}
+
+	if inp.GetDeferralReq() != nil {
+		out.Req = &v2.VulnerabilityException_DeferralReq{
+			DeferralReq: convertDeferralReq(inp.GetDeferralReq()),
+		}
+	} else if inp.GetFpRequest() != nil {
+		out.Req = &v2.VulnerabilityException_FpRequest{
+			FpRequest: &v2.FalsePositiveRequest{},
+		}
+	}
+
+	if inp.GetUpdatedDeferralReq() != nil {
+		out.UpdatedReq = &v2.VulnerabilityException_DeferralReqUpdate{
+			DeferralReqUpdate: convertDeferralReq(inp.GetUpdatedDeferralReq()),
+		}
+	}
+
+	return out
+}
+
+func convertUsers(users []*storage.SlimUser) []*v2.SlimUser {
```

Comment:
nit:  Would it make sense to put the `user` converters in a `common.go`?

Reply:
you have a point